types/map.ts

```diff
@@ -81,11 +83,26 @@ interface Map extends L.Map {
   buildings: buildings.Buildings;
   indoorMapEntities: indoorMapEntities.IndoorMapEntities;
   indoorMapFloorOutlines: indoorMapFloorOutlines.IndoorMapFloorOutlines;
-
-  // TODO
-  themes: any;
-  routes: any;
-  heatmaps: any;
+  themes: {
```

Comment:
Seems odd that methods in the themes, routes, and heatmaps modules are declared here rather than in `themes.ts` etc... is this unavoidable for some reason?

Reply:
No particular reason – moved them to separate namespaces!